### PR TITLE
Example: ammo / instancing, improve high refresh rate experience

### PR DESCRIPTION
**Pull request by CalebBabin**
Related issue: none

**Description**

Elements are "respawned" too quickly on 120/240hz monitors, this locks that process to a 60hz interval outside the render loop.

### CODE REVIEW

**Comment by CalebBabin** [comment]
Needs further adjustments.

**Comment by Mugen87** [comment]
Do you mind applying the same fix to `physics_oimo_instancing`?

File: examples/physics_ammo_instancing.html [hunk 1]
```diff
@@ -137,26 +137,28 @@
...
-				//
+					index = Math.floor( Math.random() * spheres.count );
 
-				let index = Math.floor( Math.random() * boxes.count );
+					position.set( 0, Math.random() + 1, 0 );
+					physics.setMeshPosition( spheres, position, index );
 
-				position.set( 0, Math.random() + 1, 0 );
-				physics.setMeshPosition( boxes, position, index );
+				}, 17 );
```
Comment (mrdoob) [inline examples/physics_ammo_instancing.html:155]: Hmm... Why 17 milliseconds?

Comment (Mugen87) [reply examples/physics_ammo_instancing.html:155]: I guess the OP wants to ensure that the related code block is executed 60 times per second (in order to imitate 60FPS).

Comment (mrdoob) [reply examples/physics_ammo_instancing.html:155]: Yeah... I was just curious why `17` and not `16.66` or just `1000 / 60`.

Comment (mrdoob) [reply examples/physics_ammo_instancing.html:155]: I'll merge and tweak.

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**